### PR TITLE
Added logo for AutoCaptcha

### PR DESCRIPTION
**Pull request by rahulkapoor90**
This extension is really great but it was missing a nice logo so i tried adding one, the background color of logo has color code same as that of academics portal.

@karthikb351  I also think the name should be changed from "AutoCaptcha for chrome" to something that has VIT in it as the extension is getting reviews that it is not working on other sites.

### CODE REVIEW

**Comment by karthikb351** [comment]
Oops. Haven't bothered even looking at the reviews. Good point.

Have any suggestions for a name?

**Comment by rahulkapoor90** [comment]
How about  "AutoCaptcha by/from VITAcademics", this would help the extension come up with the search results for VITacademics on google helping VITians find extension easily and also because it was made after using captcha.js in the orginal app ( i guess).

**Comment by rahulkapoor90** [comment]
@karthikb351 Why no merge?

File: autocaptcha-for-chrome/manifest.json [hunk 1]
```diff
@@ -3,17 +3,25 @@
...
+		"48": "icon-48.png",
+		"128": "icon-128.png"
+	},
   "permissions": ["*://academics.vit.ac.in/*"],
   "content_scripts": [
     {
       "matches": ["*://academics.vit.ac.in/parent/parent_login.asp","*://academics.vit.ac.in/student/stud_login.asp","*://academics.vit.ac.in/parent/","*://academics.vit.ac.in/student/","*://27.251.102.132/parent/parent_login.asp","*://27.251.102.132/student/stud_login.asp","*://27.251.102.132/parent/","*://27.251.102.132/student/"],
-      "js": ["captcha.js"]
+      "js": ["captcha.js"],
+	  "css": ["main.css"]
```
Comment (karthikb351) [inline autocaptcha-for-chrome/manifest.json]: What is main.css? :confused:

**Comment by karthikb351** [comment]
Also why not use the logo from the website? - http://karthikb351.github.io/AutoCaptcha-for-Chrome/img/favicon.ico

**Comment by rahulkapoor90** [comment]
i am working on "main.css" and what website are you talking about?

**Pull request closed without merging**